internal/connector/internal/api/dbapi/connector.go

```diff
@@ -126,12 +126,6 @@ type SchemaRegistryConnectionSettings struct {
 	Url              string
 }
 
-type ServiceAccount struct {
```

Comment:
Moved to separate file in the package as it is shared between `connector` and `processor`.